docs/conf.py

```diff
@@ -1,5 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
This is superfluous in python 3 as source files are decoded as utf-8 by default.